### PR TITLE
Upgrade SmallRye GraphQL to 1.0.5

### PR DESCRIPTION
**Pull request by phillip-kruger**
This upgrades SmallRye GraphQL to 1.0.5.

The main new feature is the `@Context` Object that is now available.

For a full list list fixes and enhancements see:
https://github.com/smallrye/smallrye-graphql/releases/tag/1.0.5

Signed-off-by: Phillip Kruger <phillip.kruger@gmail.com>

### CODE REVIEW

File: extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java [hunk 1]
```diff
@@ -129,6 +131,15 @@ void registerNativeImageResources(BuildProducer<ServiceProviderBuildItem> servic
...
                 new ServiceProviderBuildItem(LookupService.class.getName(), lookupImplementations.toArray(new String[0])));
+
+        // Schema Extension Service (We use the one from the CDI Module)
+        String schemaExtensionService = SPI_PATH + SchemaBuildingExtensionService.class.getName();
+        Set<String> schemaExtensionImplementations = ServiceUtil.classNamesNamedIn(
+                Thread.currentThread().getContextClassLoader(),
+                schemaExtensionService);
+        serviceProvider.produce(
+                new ServiceProviderBuildItem(SchemaBuildingExtensionService.class.getName(),
+                        schemaExtensionImplementations.toArray(new String[0])));
```
Comment (gsmet) [inline extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java:142]: Would it be hard to add a test for this feature?

Comment (phillip-kruger) [reply extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java:142]: Good idea. Let me look into this.

Comment (phillip-kruger) [reply extensions/smallrye-graphql/deployment/src/main/java/io/quarkus/smallrye/graphql/deployment/SmallRyeGraphQLProcessor.java:142]: Done

**Comment by gsmet** [comment]
@phillip-kruger there's a conflict now, can you resolve it? Thanks!

**Comment by phillip-kruger** [comment]
Sure, I'll resolve and then squash :)

**Review by gsmet — APPROVED** [review]

**Pull request merged**